lib/rules/block-spacing.js

```diff
@@ -14,7 +14,7 @@ const util = require("../ast-utils");
 module.exports = {
     meta: {
         docs: {
-            description: "enforce consistent spacing inside single-line blocks",
+            description: "enforce consistent spacing on the same line between an open block token and the next token, and between a closing block token and the previous token",
```

Comment:
can it be simpler? just as the title. 😄

Reply:
I don't feel strongly about the longer sentence. updated 👍